k2/python/k2/fsa.py

```diff
@@ -232,6 +235,35 @@ def __str__(self) -> str:
 
         return ans
 
+
+    def save_image(self, filename: str, title: Optional[str] = None):
```

Comment:
If this function returns the graphviz object I think it can be easily rendered inside a Jupyter notebook

Reply:
You mean return that in addition to saving the image?  Could be doable.

Reply:
Yeah. How about naming the method draw(), making it return the graphviz object and having an optional argument for the path where to save the image? Seems like a cleaner API.